src/types.ts

```diff
@@ -0,0 +1,22 @@
+import * as BN from 'bn.js'
+import { unpadBuffer } from './bytes'
+
+export type BNLike = BN | string | number
+
+export type BufferLike = Buffer | TransformableToBuffer | PrefixedHexString | number
+
+export type PrefixedHexString = string
+
+export interface TransformableToBuffer {
```

Comment:
Can we have some code comments on the new types what makes them "BN like", "Buffer like" respectively "Transformable to Buffer"?

Reply:
added in https://github.com/ethereumjs/ethereumjs-util/pull/275/commits/e48478d3449fed6142cd6f611dffab5d4ed46ee3